create.go

```diff
@@ -218,43 +216,43 @@ func createCustomResourceDefinition(namespace string, clientSet apiextensionscli
 		return nil, err
 	}
 
-	// Wait for CRD creation.
-	// Todo: Not sure if this is necesssary
-	err = wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
-		crd, err = clientSet.ApiextensionsV1beta1().CustomResourceDefinitions().Get(context.TODO(), crd.Name, metav1.GetOptions{})
-		if err != nil {
-			fmt.Printf("Fail to wait for CRD creation: %+v\n", err)
-
-			return false, err
-		}
-		for _, cond := range crd.Status.Conditions {
-			switch cond.Type {
-			case apiextensionsv1beta1.Established:
-				if cond.Status == apiextensionsv1beta1.ConditionTrue {
-					return true, err
-				}
-			case apiextensionsv1beta1.NamesAccepted:
-				if cond.Status == apiextensionsv1beta1.ConditionFalse {
-					fmt.Printf("Name conflict while wait for CRD creation: %s, %+v\n", cond.Reason, err)
-				}
-			}
-		}
-
-		return false, err
-	})
-
-	// If there is an error, delete the object to keep it clean.
-	if err != nil {
-		fmt.Println("Try to cleanup")
-		deleteErr := clientSet.ApiextensionsV1beta1().CustomResourceDefinitions().Delete(context.TODO(), crd.Name, metav1.DeleteOptions{})
-		if deleteErr != nil {
-			fmt.Printf("Fail to delete CRD: %+v\n", deleteErr)
-
-			return nil, errors.NewAggregate([]error{err, deleteErr})
-		}
-
-		return nil, err
-	}
+	// // Wait for CRD creation.
```

Comment:
Are you keeping this commented section in case it's needed?

Reply:
Yeah I'm still not clear if this is actually required or not. Let me remove for now and we can always fish it out from history if needed.